decompiler/frontend/binaryninja/handlers/constants.py

```diff
@@ -52,7 +52,8 @@ def lift_symbol(self, import_constant: mediumlevelil.MediumLevelILImport, **kwar
 
     def lift_pointer(self, constant: mediumlevelil.MediumLevelILConstPtr, **kwargs) -> Constant:
         """Helper method translating a pointer to address and binary view."""
-        return self._lift_bn_pointer(constant.constant, constant.function.source_function.view)
+        x = self._lift_bn_pointer(constant.constant, constant.function.source_function.view)
+        return x
```

Comment:
![giphy-7](https://user-images.githubusercontent.com/12004321/155492922-81584fa0-0dab-4cc1-a9cd-f76f3ce1555f.gif)

Reply:
removed debug code in 75fcf080